### PR TITLE
Make TS mechs display walk anim while turning

### LINKED ISSUE #13346: OpenRA does not consider turning on the spot a movement
In original TS, mechs played their walking anim even when turning on the spot (which makes sense), in OpenRA they don't.

### PR DESCRIPTION
**Pull request by reaperrr**
Matches the original (and looks better, too).

Closes #13346.

### CODE REVIEW

**Review by Smittytron — COMMENTED** [review]
Looks good. 
I noticed the Mammoth Mk II seems to walk while turning with or without this.

**Comment by reaperrr** [comment]
> I noticed the Mammoth Mk II seems to walk while turning with or without this.

Yeah, `WithVoxelWalkerBody` did its own thing (checking both `IsMoving` *and* whether the current facing mismatches the previous facing).

I removed the now-redundant facing check from `WithVoxelWalkerBody`, please check if the Mk2 still works as intended on this PR, just to be sure.

**Review by Smittytron — DISMISSED** [review]
Yep, works the same.

File: OpenRA.Mods.Common/Activities/Move/Move.cs [hunk 1]
```diff
@@ -225,6 +225,10 @@ public override Activity Tick(Actor self)
 			{
 				path.Add(nextCell.Value.First);
 
+				// If Mobile.Info.AlwaysConsiderTurnAsMove is true, we consider Turn as movement regardless of facing delta size.
+				// This is needed for actors that want to display their movement animation during turns (walker units, for example).
```
Comment (pchote) [inline OpenRA.Mods.Common/Activities/Move/Move.cs]: Can you please add to this comment where this gets set back to false?

**Comment by reaperrr** [comment]
Updated comment.

**Review by Mailaender — APPROVED** [review]

**Review by Mailaender — APPROVED** [review]
Works as promised.

**Review by GraionDilach — APPROVED** [review]
:+1:

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/111f3e7adea7a179effb74654ac438c3e79bcfe3...c2e459f66bebda029d6de687bcbbed9b4b41eb65

**Pull request merged**